etc/check-sse4_2.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# This script makes sure that the tikv-server executables in the target
+# directory enables intel® sse4.2 extensions. It is suitable to run as part of
+# CI.
+
+dirs="./target/debug ./target/release"
+
+errors=0
+
+# These need to check sse4.2.
+targets="tikv-server tikv-importer"
+
+if [[ "`uname`" != "Linux" ]]; then
+    echo "skipping sse4.2 check - not on Linux"
+    exit 0
+fi
+
+echo "checking bins for sse4.2"
+
+for dir in $dirs; do
+    for file in $targets; do
+        
+        dirfile="$dir/$file"
+        if [[ -x "$dirfile" && ! -d "$dirfile" ]]; then
+            
+            echo "checking binary $dirfile for sse4.2"
+            
+            # RocksDB uses sse4.2 in the `Fast_CRC32` function
+            fast_crc32=$(nm "$dirfile" | grep -o " _.*Fast_CRC32.*")
+            if [[ ! $fast_crc32 ]]; then
+                echo "error: $dirfile does not contain rocksdb::crc32c::Fast_CRC32 function"
+                errors=1
+                continue
```

Comment:
why not break here

Reply:
Because we want to check other files

Reply:
But seems once errors=1, checking other files doesn't matter the result.

Reply:
Yes, I mean we may want to know how many files are not using sse4.2.